### ISSUE TITLE
CentOS7 fails to set firewalld ports

### ISSUE THREAD

**Issue by Bluscrn** [ReinerNippes/nextcloud#104]
Running locally on CentOS7 box, firewall rules were not updated, locally corrected by adding service http,https

**Comment by ReinerNippes** [comment]
Yes. I'm afraid that's not finished yet. Only on Debian/Ubuntu the firewall is configured.

**Comment by Bluscrn** [comment]
No worries, I appreciated the project and how easy it made the setup. I just wanted to put this on your radar.
